Week1/README.md

```diff
@@ -2,33 +2,36 @@
 
 ```
 In week two we will discuss the following topics:
-• Loops (for/while)
-• Functions
-• Advanced data types [Objects]
-• Conditions (if/else statements)
-• Statements vs Expressions
+• Intro JavaScript (What is it, where can you use it for)
+• Variables [var, let, const]
+• Basic Data types [Strings, Numbers, Arrays, Booleans]
+• Operators
 • Naming conventions
 ```
 
-### Here are resources that we like you to read as a preparation for the coming lecture:
+## How to get started
+1. Download and install the latest Current version of NodeJS - from https://nodejs.org/en/download/current/
```

Comment:
Now that Node version 8 LTS is out, I would recommend that, for the time being, we ask students to install the LTS version. This already has all the ES6 features we want. Last time I checked there were still NPM packages that complained about version 9.

Reply:
yes agreed.